### PR TITLE
Add Guardian.Plug.Backdoor

### PR DESCRIPTION
**Pull request by jayjun**
@doomspork I’ve decided to port the backdoor to Guardian 1.0 only. Let me know what you think!

### CODE REVIEW

**Comment by doomspork** [comment]
@jayjun would you mind copying the basic instructions to a README?  If not I can throw that together letter 👍

**Comment by jayjun** [comment]
@doomspork Done!

**Comment by keathley** [comment]
Maybe this is silly but I'm not sure why specific instructions would be included for Hound in both the README and the module docs.

**Comment by jayjun** [comment]
@keathley Some users arrive on GitHub first, others on HexDocs. I tend to mirror the README and main module docs as much as I can.

Since testing is the raison d’être of this package, I figured it’s appropriate.

**Comment by keathley** [comment]
@jayjun Sorry, my question is really why we would include instructions specifically for Hound and not include instructions for Wallaby or some other framework?

**Comment by jayjun** [comment]
@keathley Oh, that’s grandfathered from the original pull request (that wasn’t authored by me). I’m sure the maintainers will happily accept instructions for others.

**Comment by doomspork** [comment]
@jayjun is correct 😁  The original PR this work is from was initiated because of Hound usage.  We'd welcome additions to the README for any other projects that can leverage this feature.

File: lib/guardian/plug/backdoor.ex [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  ```
+
+  NOTE: This plug is designed for acceptance testing and should never be added
+  to a production environment.
+
+  ## Usage
+
+  Now that `Guardian.Plug.Backdoor` is installed, it's time to sign in.
+
+  If you're using [Hound][hound], you can write the following code.
```
Comment (doomspork) [inline lib/guardian/plug/backdoor.ex]: Maybe we can remove the library specific examples from the moduledoc.  Instead, we can create a `Hound` section in the README.  @keathley can add his own sections later 👍 

Thoughts @jayjun?

Comment (jayjun) [reply lib/guardian/plug/backdoor.ex]: @doomspork You’re right, reference to other libraries belong in the README.

**Comment by doomspork** [comment]
Thanks @jayjun!  I will get this packaged up and released 👍

**Pull request merged**